pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -373,29 +368,14 @@ def circuit(x, y):
     >>> qml.math.allclose(result, approx, atol=1e-4)
     True
     """
-    if max_expansion is not None:
-        warnings.warn(
-            "The max_expansion argument is deprecated and will be removed in version v0.40. ",
-            qml.PennyLaneDeprecationWarning,
-        )
-    else:
-        max_expansion = 6
 
     with QueuingManager.stop_recording():
         # Build the basis set and the pipeline for intial compilation pass
         basis_set = [op.__name__ for op in _PARAMETER_GATES + _CLIFFORD_T_GATES]
         pipelines = [remove_barrier, commute_controlled, cancel_inverses, merge_rotations]
 
         # Compile the tape according to depth provided by the user and expand it
-        with warnings.catch_warnings():
-            warnings.filterwarnings(
```

Comment:
Can you remove `import warnings` from the top of this file or is it used for other things here as well?

Reply:
It's been used ...
```python
 with warnings.catch_warnings():
        warnings.simplefilter("ignore", RuntimeWarning)
        sd_ops = qml.ops.one_qubit_decomposition(
            qml.matrix(op), op.wires, "ZXZ", return_global_phase=True
        )
```